src/contentmanagerside.cpp

```diff
@@ -12,17 +12,19 @@ ContentManagerSide::ContentManagerSide(QWidget *parent) :
     mp_ui(new Ui::contentmanagerside)
 {
     mp_ui->setupUi(this);
-    connect(mp_ui->allFileButton, &QRadioButton::toggled,
-            this, [=](bool checked) { this->mp_contentManager->setLocal(!checked); });
-    connect(mp_ui->localFileButton, &QRadioButton::toggled,
-            this, [=](bool checked) { this->mp_contentManager->setLocal(checked); });
+    mp_ui->buttonGroup->setId(mp_ui->allFileButton, 0);
+    mp_ui->buttonGroup->setId(mp_ui->localFileButton, 1);
```

Comment:
Can't we set this in the ui definition  ?

Reply:
Sadly, no.